src/functions/asyncToken/requestService/requestService.ts

```diff
@@ -17,8 +16,9 @@ export class RequestService implements IProcessRequest {
       return log("Invalid authorization header");
     }
 
-    const decodeAuthorizationHeader =
-      this.decodeAuthorizationHeader(authorizationHeader);
+    const decodeAuthorizationHeader = this.decodeAuthorizationHeader(
+      authorizationHeader!,
```

Comment:
We can avoid using `!` if the validation isn't in a private function.

```   
if (!authorizationHeader) {
  return log("Missing authorization header");
}

if (!authorizationHeader.startsWith("Basic ")) {
  return log("Invalid authorization header");
}
```
But confident with it remaining as is